### PR TITLE
Admin panel

### PR DESCRIPTION
**Pull request by Amaimon208**

### CODE REVIEW

File: src/main/java/com/candyshop/islodycze/registration/WebSecurityConfig.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ protected void configure(AuthenticationManagerBuilder auth) {
     protected void configure(HttpSecurity http) throws Exception {
         http.authorizeRequests()
                 .antMatchers("/users").hasAuthority("ADMIN")
-                .and().authorizeRequests().antMatchers("/shopping_cart").hasAuthority("USER")
+                .and().authorizeRequests().antMatchers("/shopping_cart").permitAll()
```
Comment (P1200) [inline src/main/java/com/candyshop/islodycze/registration/WebSecurityConfig.java]: There should be permit only for `USER` and `ADMIN`

File: src/main/resources/templates/MainPage/admin_panel.html [hunk 2]
```diff
@@ -8,38 +8,24 @@
...
-        </ul>
-        <div class="col-8">
-            <input type="text" th:field="${productSearchCriteria.searchedPhrase}"/>
-            <button type="submit" class="btn btn-primary mb-1">Wyszukaj</button>
-            <a th:href="@{'/?productName='}" class="btn btn-primary mb-1">Wyczysc</a>
-        </div>
+            <a th:href="@{/add_productForm}" class="btn btn-primary mb-2">Dodaj produkt</a>
 
-        <span th:if="${top10List != null}">
+            <!--    !!!!AfterMergingPDF PR !!!-->
```
Comment (P1200) [inline src/main/resources/templates/MainPage/admin_panel.html]: Commented code.

File: src/main/resources/templates/add_category.html [hunk 3]
```diff
@@ -14,18 +14,33 @@
...
+        </tr>
+        </thead>
+        <tbody>
+        <tr th:each="category: ${categoryList}">
+            <td>
+                <label th:text="${category.categoryName}"></label>
+            </td>
+            <td> <a method=delete th:href="@{'/deleteCategory/'+${category.categoryName}}"
+                    class="btn btn-primary mb-2">
+                    Usun Kategorie</a></td>
```
Comment (P1200) [inline src/main/resources/templates/add_category.html]: change `<meta charset="ISO-8859-1">` to `<meta charset="UTF-8">` and add polish chars.

**Review by P1200 — CHANGES_REQUESTED** [review]
Pls add some screen shots of your changes.

File: src/main/resources/templates/add_category.html (hunk 3 above)
Comment (Amaimon208) [reply src/main/resources/templates/add_category.html]: Fixed

File: src/main/resources/templates/MainPage/admin_panel.html (hunk 2 above)
Comment (Amaimon208) [reply src/main/resources/templates/MainPage/admin_panel.html]: Fixed

File: src/main/java/com/candyshop/islodycze/registration/WebSecurityConfig.java (hunk 1 above)
Comment (Amaimon208) [reply src/main/java/com/candyshop/islodycze/registration/WebSecurityConfig.java]: Fixed

**Review by P1200 — APPROVED** [review]

**Pull request merged**